src/Assets/app.js

```diff
@@ -1,15 +1,28 @@
 import { initAll } from 'govuk-frontend';
+import CookieMessage from './Javascript/cookie-message';
 import BackLink from './Javascript/back-link';
-import './Javascript/cookie-bar.js';
-import './Javascript/accordion.js';
-import './Javascript/toggle.js';
+import Accordion from './Javascript/accordion';
+import Toggle from './Javascript/toggle';
 import './Javascript/typeahead.jquery.js';
 import './Styles/site.scss';
 
 initAll();
 
-var $backLink = document.querySelector('[data-module="back-link"]')
-new BackLink($backLink).init()
+var $cookieMessage = document.querySelector('[data-module="cookie-message"]');
+new CookieMessage($cookieMessage).init();
```

Comment:
All this boilerplate initialisation would be improved with a pattern like https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules